commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -86,16 +85,17 @@ class Group(StrictJsonObject):
     vars = jsonobject.DictProperty()
 
 
-def provision_machines(spec, env=None):
-    if env is None:
-        env = u'hq-{}'.format(
+def provision_machines(spec, env_name=None):
+    if env_name is None:
+        env_name = u'hq-{}'.format(
             ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(7))
         )
-    inventory = bootstrap_inventory(spec, env)
-    instance_ids = ask_aws_for_instances(env, spec.aws_config, len(inventory.all_hosts))
+    environment = get_environment()
```

Comment:
shouldn't this have `env_name` as a parameter?

Reply:
Sure should!